lib/Extension/Bundle/Traits/StateHelperTrait.php

```diff
@@ -27,39 +27,13 @@
  */
 trait StateHelperTrait
 {
-    /**
-     * @deprecated will be removed in Pimcore 11
-     *
-     * Helper method to check enabled state from class definitions/non-service instances
-     *
-     * @return bool
-     */
-    public static function isEnabled(): bool
-    {
-        $bundleManager = \Pimcore::getContainer()->get(PimcoreBundleManager::class);
-
-        if (!$bundleManager->exists(__CLASS__)) {
-            return false;
-        }
-
-        if (!$bundleManager->isEnabled(__CLASS__)) {
-            return false;
-        }
-
-        return true;
-    }
-
     /**
      * Helper method to check installed state from class definitions/non-service instances
      *
      * @return bool
      */
     public static function isInstalled(): bool
     {
-        if (!static::isEnabled()) {
-            return false;
-        }
-
```

Comment:
Should we move `isInstalled()` method to the `AbstractPimcoreBundle` class, which in turn injects the bundle manager?